app/src/main/java/de/danoeh/antennapod/fragment/QueueFragment.java

```diff
@@ -443,6 +443,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         registerForContextMenu(recyclerView);
 
         SwipeRefreshLayout swipeRefreshLayout = root.findViewById(R.id.swipeRefresh);
+        swipeRefreshLayout.setDistanceToTriggerSync(500);
```

Comment:
Why is it sometimes 300 and sometimes 500? I think it should be consistent on all screens. Storing it in `integers.xml` and querying with `getResources().getInteger(R.integer.swipe_refresh_distance)` would ensure that it stays consistent.

Reply:
When using multi window, in ```FeedItemListFragment``` pull to refresh is not working for any value greater than 300 and on other screens it does not have much resistance so I used 500. To be consistent, changed value to 300 for all screens.